packages/react-router-dom/__tests__/special-characters-test.tsx

```diff
@@ -828,19 +828,8 @@ describe("special character tests", () => {
     });
 
     describe("browser routers", () => {
-      let testWindow: Window;
-
-      beforeEach(() => {
-        // Need to use our own custom DOM in order to get a working history
-        const dom = new JSDOM(`<!DOCTYPE html><html><body></body></html>`, {
-          url: "https://remix.run/",
-        });
-        testWindow = dom.window as unknown as Window;
-        testWindow.history.pushState({}, "", "/");
-      });
```

Comment:
Reusing the same window instance across these tests was. ... not smart on my part :)